src/azul/service/manifest_service.py

```diff
@@ -224,6 +224,17 @@ def _create_name_object(self, manifest_object_key, base_name, extension):
                                  data=file_name.encode())
         return file_name
 
+    def fetch_cached_manifest(self, format_: str, filters: Optional[str] = None):
+        filters = self.parse_filters(filters)
+        generator = ManifestGenerator.for_format(format_, self, filters)
+        hash_code = generator.manifest_hash_code
+        manifest_key = self._derive_manifest_key(filters, hash_code, format_)
+        object_key = f'manifests/{manifest_key}.{generator.file_name_extension}'
+        if generator.is_cacheable and self._can_use_cached_manifest(object_key):
+            file_name = self._use_cached_manifest(generator, object_key)
+            return self.storage_service.get_presigned_url(object_key, file_name=file_name)
+        return False
```

Comment:
:else branch. Please remember this guideline. It's mentioned in the contributing guide.